src/components/notification/notification.vue

```diff
@@ -25,6 +42,26 @@ module.exports = {
       default: false,
       type: Boolean
     }
+  },
+  computed: {
+    modifierClass() {
+      const modifiers = [];
+      if (this.type) {
+        modifiers.push(`apos-notification--${this.type}`);
+      }
+      if (modifiers.length > 0) {
```

Comment:
So far this doesn't seem like we need to go through the array stuff. We can only have one class as this is written.

Reply:
well going to put the base class in as per another comment and keep, i like the future flexibility